### PR TITLE
Added Hebrew translation.

### PR DESCRIPTION
**Pull request by yarons**
## PR Description:

Adding base.po and base.mo in Hebrew (Full translation).

## Tests and Checks
- [x] I have tested the code!<be>

Tested on a virtual machine and found several issues described in #2347, but having the complete translation will motivate fixing those :)

### CODE REVIEW

**Comment by yarons** [comment]
There's some error regarding a missing license file. Is there anything I can do to fix it?

**Comment by yarons** [comment]
I've accidentally added the README change to this branch, I'll start over if needed, sorry about that.

File: README.md [hunk 1]
```diff
@@ -27,9 +27,7 @@ Assuming you are on an Arch Linux live-ISO or installed via `pip`:
 
 ## Running the [guided](https://github.com/archlinux/archinstall/blob/master/archinstall/scripts/guided.py) installer using `git`
 
-    # cd archinstall-git
```
Comment (svartkanin) [inline README.md:30]: Can we leave this line as it's necessary

**Review by svartkanin — CHANGES_REQUESTED** [review]

**Comment by svartkanin** [comment]
That's okay, I left a small comment

**Comment by svartkanin** [comment]
Great, I don't have merging power so we have to wait for @Torxed

**Review by Torxed — APPROVED** [review]

**Comment by yarons** [comment]
I've observed some issues with the Linux native console (the text is reversed), fbterm looks better but I'm looking for a better-supported console solution.

**Pull request merged**